### DISCUSSION TITLE
Jamulus on iOS

### DISCUSSION THREAD

**Discussion by RobyDati** [jamulussoftware/jamulus#2263] [Technical Support & Help]
Hi, I followed the instructions on https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md#ios but I am stacked on step 5:   I succeeded to compile with Xcode v 13.2.1 with qt 5.12.2 and with destination simulator and get the Jamulus run on simulator of my iPhone 11 iOS 15.2. The problems on Signing & Capabilities did not stop the build, they are just yellow warnings.   But when I try to connect my iPhone device and build to it the Signing errors are red and build fail from start.  I registered to the Apple Developer Free, and in Xcode I set the following:   Automatically manage signing. On,   Team : Roberto Dati (Personal Team),  Bundle Identifier io.jamulus.Jamulus, Provisioning Profile: Xcode Managed Profile,  Signing Certificate: Apple Development.    Errors are "Failed to register bundle identifier.    No profiles for "io.jamulus.Jamulus" were found.     I tried also to add Account GitHub in Xcode but it doesn't help.   Is there a way to set up correctly the Signing & Capabilities tab  in Xcode?

**Comment by ann0see** [comment]
Yes. You need to change the bundle identifier too. I think it’s somewhere in the main tab? If you succeed, could you please document this step in the respective file?
https://github.com/jamulussoftware/jamulus/edit/master/COMPILING.md

**Reply by RobyDati** [reply]
Thank you, I succeded and Jamulus app is running on my iPhone thanks to your suggestion to modify the bundle identifier under the General Tab in Xcode. Then step 5 in https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md#ios could be more detailed this way:                                       5. Go to the Signing & Capabilities tab and fix signing errors by setting a team. Xcode might tell you if you need to change anything else like the bundle id. For instance, you can use your free Apple Developer Program Account as a Personal Team, and set it up under Xcode Menu->Preferences->Accounts. Then choose a Bundle Identifier at your choice in the relevant field under General Tab in section Identity. Then go to Signing & Capabilities tab, section Identity, the Automatic manage signing option and you should get Team: (Your Name) (Personal Team), Bundle identifier: (the same you modified on General Tab), Provisioning Profile: Xcode Management Profile, Signing Certificate: Apple Development (your e-mail used for signing in to Apple)

**Reply by ann0see** [reply]
Great! Thanks. Good to hear that. Could you please update the file yourself (this gives you the credit)?

**Reply by RobyDati** [reply]
OK, I updated the file https://github.com/jamulussoftware/jamulus/edit/master/COMPILING.md , it's first time I do this operation in github, I hope I've done it correctly, thank you again.

**Reply by ann0see** [reply]
Great! I can see it’s on your fork. Could you please select the patch-1 branch on your repo: https://github.com/RobyDati/jamulus/tree/patch-1

and Click on contribute to open a Pull request to this repository?

**Reply by RobyDati** [reply]
OK I followed the steps to open a Pull request as per your indications. It's my first time doing these tasks. Thank you
